packages/netlify-cms-core/src/actions/__tests__/config.spec.js

```diff
@@ -634,7 +634,7 @@ describe('config', () => {
         ).toEqual({ structure: 'multiple_folders', locales: ['en', 'fr'], default_locale: 'fr' });
       });
 
-      it('should throw when i18n is set on files collection', () => {
+      it('should throw when i18n structure is not single_file on files collection', () => {
```

Comment:
Can we explicitly test for an error being thrown when structure is one of `multiple_folders, multiple_files`?